app/src/app/pages/plando-page/plando-page.component.ts

```diff
@@ -127,6 +136,16 @@ export class PlandoPageComponent implements OnInit {
     return { invalidPlandoItem: { value: control.value } };
   }
 
+  public resetPlandoForm() {
+    if (confirm('This will reset all plando settings. Are you sure?')) {
+      this.formGroup.reset();
+      for (const power of STAR_SPIRIT_POWER_NAMES) {
+        this.formGroup.get('move_costs').get('starpower').get(power).setValue(-1);
```

Comment:
Why is the reset not good enough for these?

Reply:
The material sliders default to the `0` position when reset, even if their default form control value and/or the attribute value is set to -1 (I tried both). Without this line, the sliders end up in an inconsistent state on reset, showing `?` when they're at the `0 SP` position:
![image](https://github.com/user-attachments/assets/84b563f3-d695-4fb8-9d19-5d6e259e423d)